src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -29,31 +28,21 @@ public class ServerJob : IIdentifiable
         [JsonConverter(typeof(StringEnumConverter))]
         public KestrelTransport? KestrelTransport { get; set; }
 
-        [JsonConverter(typeof(StringEnumConverter))]
-        public Scenario Scenario { get; set; }
+        public string Scenario { get; set; }
 
         [JsonConverter(typeof(StringEnumConverter))]
         public Scheme Scheme { get; set; }
+        public int Port { get; set; } = 5000;
+        public string Path { get; set; }
         public string AspNetCoreVersion { get; set; }
         
         // Delay from the process started to the console receiving "Application started"
         public TimeSpan StartupMainMethod { get; set; }
-        // Delay from the process started to the result of the first request
-        public TimeSpan StartupFirstRequest { get; set; }
-        public TimeSpan Latency { get; set; }
         public List<ServerCounter> ServerCounters { get; set; } = new List<ServerCounter>();
-        private IEnumerable<Source> _sources;
-        public IEnumerable<Source> Sources
-        {
-            get
-            {
-                return _sources ?? Enumerable.Empty<Source>();
-            }
-            set
-            {
-                _sources = value;
-            }
-        }
+        public IList<Source> Sources { get; set; } = new List<Source>();
+
+        public Source Source { get; set; } = new Source();
```

Comment:
Rename either `Sources` or `Source` to clarify the difference, or at least add a comment.